apps/epic-web/src/pages/api/progress.ts

```diff
@@ -18,6 +18,7 @@ const lesson = async (req: NextApiRequest, res: NextApiResponse) => {
         lessonProgress.map((progress) => {
           return {
             lessonId: progress.lessonId,
+            lessonSlug: progress.lessonSlug,
```

Comment:
we specifically only store the lessonId (it's Sanity id) and don't use slugs for recording progress. we've found that it is more stable and gives learners a more consistent progress experience. lesson/module/section slugs are still in the data model for legacy (lazy) reasons, but aren't used.

Can you let me know what you're thinking here?

Reply:
That makes sense. The reason I thought to do this is because the API for getting modules includes a lesson "solution" which doesn't have an ID. If you merge #1198 then I think that should work.